Sources/Signal.swift

```diff
@@ -426,6 +426,12 @@ extension Signal: SignalProtocol {
 	}
 }
 
+extension Signal: SignalProducerRepresentable {
+	public var producer: SignalProducer<Value, Error> {
+		return SignalProducer(self)
```

Comment:
`Signal<Signal>` overloads are all implemented by lifting the inner `Signal`s to `SignalProducer`.